Modules/Core/Common/test/itkImageRandomIteratorTest2.cxx

```diff
@@ -22,16 +22,17 @@
 #include "itkImageFileWriter.h"
 #include "itkImageFileReader.h"
 #include "itkTestingComparisonImageFilter.h"
+#include "itkTestingMacros.h"
 
 
 int
 itkImageRandomIteratorTest2(int argc, char * argv[])
 {
   if (argc < 2)
   {
-    std::cerr << "Missing arguments " << std::endl;
-    std::cerr << "Usage: " << std::endl;
-    std::cerr << argv[0] << "  outputImageFile" << std::endl;
+    std::cerr << "Missing parameters." << std::endl;
```

Comment:
The original term `arguments` sounds fine to me, why do you prefer `parameters` here? Although both terms are used interchangeably, "arguments" typically refer to the actual values passed to a function (as in this case), while "parameters" typically refer to the names listed in the function definition. https://developer.mozilla.org/en-US/docs/Glossary/Parameter

But I very much like your proposal to use `itkNameOfTestExecutableMacro(argv)`  😃

Reply:
@N-Dekker It is not a matter of preference, it is what the SW Guide says:
https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/blob/master/SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex#L2853
If you think that the term is technically inaccurate, please make a PR to the SW Guide, and a subsequent sweeping change on the ITK code base can follow in a PR separate to this one.

In another PR I brought up the idea of putting these checks and the end of test message+return statement (I believe the message is helpful, but it has been a matter of discussion) in two functions re-usable across the code to avoid these boilerplate statements and make things really consistent. But I do not have the bandwidth to implement that at this time.

Reply:
@N-Dekker I'm marking this as resolved. Please feel free to propose a PR as time permits. Thanks for the review :+1: .